recipe/meta.yaml

```diff
@@ -32,24 +32,26 @@ requirements:
     - {{ compiler('c') }}
     - {{ stdlib("c") }}
   host:
-    - python         # [not use_noarch]
-    - python >=3.8   # [use_noarch]
+    - python                   # [not use_noarch]
+    - python {{ python_min }}  # [use_noarch]
     - pip
     - cython
-    - setuptools
+    - setuptools >=46.4.0
   run:
     - aiohappyeyeballs >=2.3.0
-    - python         # [not use_noarch]
-    - python >=3.8   # [use_noarch]
+    - python                     # [not use_noarch]
+    - python >={{ python_min }}  # [use_noarch]
     - aiosignal >=1.1.2
-    - async-timeout >=4.0,<5.0  # [py<311]
+    - async-timeout >=4.0,<5.0   # [py<311 or use_noarch]
```

Comment:
This constraint is needed for both python<311 and noarch: python in order for `pip check` to pass.